fifo/from.go

```diff
@@ -49,8 +49,9 @@ func (f *From) Sync(ctx context.Context) (int64, int64, error) {
 		if err != nil {
 			return 0, 0, err
 		}
-		batchLen := batch.Length()
-		if batchLen == 0 {
+		vals := batch.Values()
```

Comment:
In the Zed repository, this variable is always called `zvals` (in case you want to go for consistency).

Reply:
as discussed